iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTagExec.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.mr.hive;
+
+import org.apache.hadoop.hive.ql.parse.AlterTableTagSpec;
+import org.apache.iceberg.ManageSnapshots;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.util.SnapshotUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class IcebergTagExec {
+
+  private static final Logger LOG = LoggerFactory.getLogger(IcebergTagExec.class);
+
+  private IcebergTagExec() {
+  }
+
+  public static void createTag(Table table, AlterTableTagSpec.CreateTagSpec createBranchSpec) {
```

Comment:
looks like a copy-paste: createBranchSpec -> createTagSpec. How different are the 2 spec classes?

Reply:
that's fine, param names a copy-pasted, see createBranchSpec -> createTagSpec

Reply:
Have fixed in commit https://github.com/apache/hive/commit/6ef586f028f1d14a7cfbb21fef5b881d48d694a8.